### DISCUSSION TITLE
为啥 studiopanel 里面需要区分是否有 logo||action 来判断是否传入 classname 啊

### DISCUSSION THREAD

**Discussion by prgrmrwy** [alibaba/designable#152] [Q&A] [answered]
https://github.com/alibaba/designable/blob/a9943dd83130b9f2bec933530a30537af13ce025/packages/react/src/panels/StudioPanel.tsx?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L42

**Comment by janryWang** [comment, marked as answer]
bug，提个PR修一下
